vendor/github.com/hlandau/passlib/README.md

```diff
@@ -0,0 +1,95 @@
+passlib for go
```

Comment:
I'm not sure I'm following, I just ran:
`glide get github.com/hlandau/passlib`
`glide get github.com/dgrijalva/jwt-go`

Is there anything else I should pay attention to ?